src/modules/leveling/voice/voice.js

```diff
@@ -25,7 +25,10 @@ export default async function voiceStateUpdate(client, oldState, newState) {
     if (newState.guild.id !== config.defaultGuild || oldState.member.user.bot) return;
     const userDto = userDb.get(newState.member.id);
     const now = Date.now();
-    const xpToGive = ((now - userDto.voiceTimeStampJoin) / 1000) * userDto.voiceXpMultiplier;
+    const xpToGive = Math.max(
```

Comment:
Shouldn't this be `Math.min()`
```suggestion
    const xpToGive = Math.min(
```